### ISSUE TITLE
Mixed Content XMLHttpRequest at /settings

### ISSUE THREAD

**Issue by lanrat** [Tautulli/Tautulli#700]
- [ ] Version
  1.3.16 
- [ ] Branch
  master
- [ ] Commit hash
  00c0c96b1b898d325b1babb8e120e8ec88892e2b
- [ ] Operating system
  Linux 3.16.0-4-amd64
- [ ] Python version
  2.7.11 (default, Dec 15 2015, 16:46:19) [GCC 4.8.4]
- [ ] What you did?
  Running plexpy behind nginx reverse proxy which handles SSL
  `[Browser] ---- (HTTPS) ---- [NGINX] ---- (HTTP) ---- [PLEXPY]`
  Clicked save in the plexpy setting page after changing the hostname of my plex server
- [ ] What happened?
  The browser console threw the following error:

```
Mixed Content: The page at 'https://domain.tld/plexpy/settings#tab_tabs-5' was loaded over HTTPS, but requested an insecure XMLHttpRequest endpoint 'http://domain.tld/plexpy/settings'. This request has been blocked; the content must be served over HTTPS.
```
- [ ] What you expected?
  For plexpy to use the same scheme it was served as so not be blocked for mixed content.
- [ ] How can we reproduce your issue?
  Run Plexpy behind a reverse proxy that handles SSL.
  I can provide my nginx configuration if needed but any reverse proxy with SSL should behave similarly.
- [ ] What are your (relevant) settings?
  "Enable HTTPS" is not enabled for plexpy because nginx handles the SSL.

**Comment by JonnyWong16** [comment]
Fixed in the dev branch 955b69a9bf7472bd3d70ae922be7eb83f5997d04

**Comment by JonnyWong16** [comment]
Fixed in v1.4.0.
